### DISCUSSION TITLE
Is it possible to change the PB Logo

### DISCUSSION THREAD

**Discussion by Jonas-Cardoso** [pocketbase/pocketbase#870] [Q&A]
I'd like to know if there's a way to change the pocketbase logo, in the app & auth pages something custom.

**Comment by ganigeorgiev** [comment]
No, at the moment this is not possible without modifying the source.

In the future there are plans to provide some extensibility to the prebuilt Admin UI but for now it's a low priority and may not be added for v1.0.0.
